### DISCUSSION TITLE
Can pyvsc be used with a 32 bits python version?

### DISCUSSION THREAD

**Discussion by fdrichardson** [fvutils/pyvsc#157] [Q&A]
I would like to use pyvsc and cocotb with modelsim as my simulator. When I do so with  a 32 bits version of python pyboolector doesn't seems to be installed properly.

**Comment by mballance** [comment]
Hi @fdrichardson,
    As you noticed, currently only 64-bit Linux builds of pyBoolector are available. You don't mention the platform (ie Windows/Linux) above. Are you using Linux?
    I ask because, if you happen to be using the OEM version of Modelsim provided with Quartus, the latest Quartus version provides a 64-bit version of Modelsim. You do have to request a free license, but I found the process pretty simple overall.

**Reply by fdrichardson** [reply]
Hello @mballance, I am using Linux. Unfortunately I am not using the OEM version of Modelsim provided with Quartus. Thank you for your quick response.
